resources/config/config.php

```diff
@@ -176,5 +176,17 @@
             'min'           => 200,
             'step'          => 50
         ]
-    ]
+    ],
+    'word_wrap' => [
+        'type'     => 'anomaly.field_type.select',
+        'required' => true,
+        'config'   => [
+            'options'       => [
+                'yes' => 'Yes',
```

Comment:
We have to use language keys: https://github.com/anomalylabs/streams-platform/blob/1.0/develop/resources/lang/en/misc.php#L4